app/use_cases/calculate_scores.rb

```diff
@@ -123,6 +125,13 @@ def perform_operation_average(data)
     data.sum(0.0) / data.size
   end
 
+  def perform_operation_sum(data)
+    raise MyMissingDataError, 'trying to calculate the sum of an empty array' if data.size.zero?
+    return data[0] if data.size == 1 # no need to convert to float if we have just one integer
+
+    data.sum(0)
```

Comment:
do we need to convert to float otherwise? or is that taken care of by sum?

Reply:
Nope there is no need to convert to float here. If one of the values in a calculation is a Float, ruby will automatically convert the whole thing to a float. And otherwise we can just stay as an integer (which is the same as when rounding to 0 decimals).